packages/eslint-plugin-nimble/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@ni-private/eslint-plugin-nimble",
+  "version": "0.0.1",
```

Comment:
Do you or @rajsite know how versioning will work for local private packages? I guess beachball won't be involved since it isn't published anywhere? So do we ever need to change this version number or the number listed in packages that depend on this?

Reply:
Yea doesn't change, just needs to be defined. We mark our other private package "version": "1.0.0" so we should do the same

```suggestion
  "version": "1.0.0",
```